qutip/core/cy/coefficient.pyx

```diff
@@ -745,7 +745,7 @@ cdef class ConstantCoefficient(Coefficient):
     """
     cdef complex value
 
-    def __init__(self, complex value):
+    def __init__(self, complex value, **_):
```

Comment:
What is the `**_` for here? The other coefficient subclasses don't accept arbitrary additional keyword arguments.

Reply:
Yes, they all do.

It's for the coefficient function. It call every `???Coefficient.__init__` with the same call: https://github.com/qutip/qutip/blob/418e33807b3f63a6d2879dad792f85b500dfc855/qutip/core/coefficient.py#L189